WcaOnRails/app/views/results_submission/submit.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide(:title, "Results submission") %>
+<%= render layout: 'nav' do %>
+  <h1><%= yield(:title) %></h1>
+
+  <p>Thank you for submitting the results!</p>
```

Comment:
I think that a separate view is not necessary for this, you could simply redirect to the competition page with a successful flash message, would this make sense?